keyboards/zfrontier/big_switch/keymaps/default/keymap.c

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2020 zFrontier
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(KC_ENTER),
+};
+
+
+
+void keyboard_post_init_user(void) {
+    rgblight_enable_noeeprom();
+    rgblight_mode_noeeprom(RGBLIGHT_MODE_BREATHING);
+    rgblight_setrgb(128,0,0);
+}
+
+#ifdef TAP_DANCE_ENABLE
+qk_tap_dance_action_t tap_dance_actions[0];
+#endif
```

Comment:
I think if you add a rules.mk to this keymap containing just `TAP_DANCE_ENABLE = no` (or disable it at the keyboard level and enable it only for the `longtap` keymap), this block can be deleted.

Reply:
Good suggestion!

Reply:
removed